src/App.vue

```diff
@@ -1,16 +1,37 @@
 <template>
   <div id="app">
     <TheHeader />
-    <router-link to="/" />
+    <router-link to="/" exact/>
+    <router-link to="/results/item-details" exact />
+    <!-- make dynamic with :name from park object -->
+      <ResultsItemDetails @save:park='savePark' />
+    <!-- </router-link> -->
     <router-view/>
   </div>
 </template>
 
 <script>
 import TheHeader from './components/TheHeader.vue'
+import ResultsItemDetails from './components/ResultsItemDetails.vue'
 
 export default {
-  components: { TheHeader }
+  components: { TheHeader, ResultsItemDetails },
+  data() {
+    return {
+      savedParks: []
+    }
+  },
+  provide() {
+    return {
+      savedParks: this.savedParks,
```

Comment:
I added `savedParks` because we will need it eventually.  I figure keeping this data on the parent-most component (`App`) would lead to more flexible access.  Any thoughts on this?